packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -2,30 +2,97 @@
     <div
         :class="$style['c-rating']"
         data-test-id="rating-component">
-        {{ copy.text }}
+        <ul :class="$style['c-rating-star-wrapper']">
+            <li
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="$style['c-rating-star']">
+                <star-filled-icon
+                    v-if="hasRating(star)"
```

Comment:
I think this might be a little confusing. So a star in this case is just a number less than or equal to the `maxStarRating`

If maxStarRating is 5, and the current key is 3 then the code reads like:

`v-if="hasRating(3)"` - is that right? 

Sorry, I just find it a little hard to read (I might need a coffee) :)

Reply:
I wonder if something like `v-if="shouldFillStar(index)"` might be clearer

Reply:
Ill change it to `hasRatingStarFilled`.  So it matched the class somewhat.